android/src/main/res/values/strings.xml

```diff
@@ -39,6 +40,9 @@
     <string name="return_button">Return</string>
     <string name="unknown">Unknown</string>
     <string name="unknown_error">Unknown error</string>
+    <string name="no_identities">You don\'t have an account yet
+
```

Comment:
No empty lines in strings.xml files. You can use `\n` for inserting new lines. Or if you will parse it as HTML, then `<br/>`.